src/System.Numerics.Vectors/src/System/Numerics/Matrix4x4.cs

```diff
@@ -1824,26 +1852,33 @@ public static Matrix4x4 Lerp(Matrix4x4 matrix1, Matrix4x4 matrix2, float amount)
         /// <returns>The negated matrix.</returns>
         public static Matrix4x4 Negate(Matrix4x4 value)
```

Comment:
Not sure if you saw, since the thread is now "hidden" by GitHub, but I commented [here](https://github.com/dotnet/corefx/pull/31779#discussion_r210401298) (with disassembly using current head of CoreCLR) that the JIT does the right thing for inlining `public static Matrix4x4 Negate(Matrix4x4 value) => -value`

Reply:
yeah it's hard to follow when github collapses them 🙂